kratos/solving_strategies/schemes/residual_based_bossak_velocity_scheme.h

```diff
@@ -563,7 +563,7 @@ class ResidualBasedBossakVelocityScheme : public Scheme<TSparseSpace, TDenseSpac
         {
             Element& r_element = *(rModelPart.ElementsBegin() + i);
             Geometry<Node<3>>& r_geometry = r_element.GetGeometry();
-            DerivativesExtension& r_extensions = *r_element.GetValue(DERIVATIVES_EXTENSION);
+            SchemeExtension& r_extensions = *r_element.GetValue(SCHEME_EXTENSION);
```

Comment:
this definitely does not belong to the base scheme!! we do not want to set a variable for every element.

also as far as i could see it, it was not used in the base scheme

Reply:
It is used to get the derivative variables from the element.